### ISSUE TITLE
Bat-suit is greyed out in Batman: Arkham Origins.

### ISSUE THREAD

**Issue by tanvmaht10** [italomandara/CXPatcher#193]
Tested in Latest Crossover with CXPatcher and the color of the bat-suit is not true black.

**Comment by italomandara** [comment]
if this happens with dxvk try launching the game with this env variable
`NAS_DISABLE_UE4_HACK=1`

**Comment by tanvmaht10** [comment]
Doesn't work

**Comment by italomandara** [comment]
> Doesn't work

Have you tried d3dmetal?
Have you tried with original crossover? (not patched)

**Comment by tanvmaht10** [comment]
Yes tried with D3DMetal and it still remains the same. Original Crossover can't run the game.

**Comment by italomandara** [comment]
If it happens on d3dmetal then we're out of options, it could be a metal issue.

**Comment by tanvmaht10** [comment]
Whether it be WineD3D,DXVK or D3DMetal, the issue persists.

**Comment by italomandara** [comment]
> Whether it be WineD3D,DXVK or D3DMetal, the issue persists.

This doesn't seem a CXPatcher issue, since it also affects d3dmetal
